source/Public/SQL Database/Get-FabricSQLDatabase.ps1

```diff
@@ -2,15 +2,15 @@ function Get-FabricSQLDatabase {
 
     <#
 .SYNOPSIS
-    Retrieves Fabric SQLDatabases
+    Returns details about Fabric SQL Databases
```

Comment:
In most of the cases (252 vs 26) we use "Retrieves" instead of "Returns". For consistency, we may consider using one or the other. Low priority, though.

Reply:
Powershell returns objects this is the language that is commonly used.  
Happy for you to start a discussion